api/routes/test.ts

```diff
@@ -1,30 +1,28 @@
-import { FastifyPluginCallback, FastifyRequest } from 'fastify';
+import { ObjectId } from '@fastify/mongodb';
+import { FastifyPluginCallback } from 'fastify';
 
 export const testRoutes: FastifyPluginCallback = (fastify, _options, done) => {
   const collection = fastify.mongo.db?.collection('user');
 
-  fastify.get('/test', async (_request, _reply) => {
+  fastify.addHook('onRequest', (req, res, done) =>
+    fastify.authenticateSession(req, res, done)
+  );
```

Comment:
```suggestion
  fastify.addHook('onRequest', fastify.authenticateSession);
```
Same argument as `fastify.authenticate`